src/panel/board.ts

```diff
@@ -111,8 +111,17 @@ export class Board {
 
         if (Util.isIn(tilePos.x, tilePos.y, 0, 0, this.tileWidth, this.tileHeight)) {
             // Hovering on a tile, so highlight it
-            // Return a callback that can both hover, and un-hover the targeted tile
             const tile: Tile = this.tileAt(tilePos.x, tilePos.y);
+            
+            // Update info box
+            this.game.infoPanel.infoBox.updateInfo_color_pos_text_header(
```

Comment:
This should be inside `onStartHover()`, no? Then it will get properly cleared if you move your cursor off the game board (by also sending an update in `onEndHover()`